qt/applications/workbench/workbench/plotting/functions.py

```diff
@@ -218,7 +235,10 @@ def pcolormesh(workspaces, fig=None):
         if subplot_idx < workspaces_len:
             ws = workspaces[subplot_idx]
             ax.set_title(ws.name())
-            pcm = ax.pcolormesh(ws, cmap=DEFAULT_CMAP)
+            if use_imshow(ws):
+                pcm = ax.imshow(ws, cmap=DEFAULT_CMAP, aspect='auto')
```

Comment:
Not a good idea. What if the user relies on getting a pcolormesh back?

Reply:
Where would this be called outside of the GUI?